src/ViewLayer/RaceModeDisplay/RaceModeDashboardView.cpp

```diff
@@ -74,6 +75,14 @@ RaceModeDashboardView::RaceModeDashboardView(BatteryPresenter& batteryPresenter,
     connect(faultsTimer_.data(), SIGNAL(timeout()), this, SLOT(updateMotor0Faults()));
     connect(faultsTimer_.data(), SIGNAL(timeout()), this, SLOT(updateMotor1Faults()));
     faultsTimer_->start(FAULT_UPDATE_PERIOD);
+    fadeEffect_ = new QGraphicsColorizeEffect(&(ui_.raceModeDashboard()));
+    fadeEffect_->setColor(QColor(0, 0, 0));
```

Comment:
I think to clean up this constructor, initializing the animation should be extracted to a private helper